File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/Avvik.java
package no.nav.syfo.controller.domain.sykepengesoknad;

import java.io.Serializable;

public class Avvik implements Serializable {
    public Double arbeidstimerNormalUke;
    public Integer arbeidsgrad;
    public Double timer;
    public Integer beregnetArbeidsgrad;

    public Avvik withArbeidstimerNormalUke(final Double arbeidstimerNormalUke) {
        this.arbeidstimerNormalUke = arbeidstimerNormalUke;
        return this;
    }

    public Avvik withArbeidsgrad(final Integer arbeidsgrad) {
        this.arbeidsgrad = arbeidsgrad;
        return this;
    }

    public Avvik withTimer(final Double timer) {
        this.timer = timer;
        return this;
    }

    public Avvik withBeregnetArbeidsgrad(final Integer beregnetArbeidsgrad) {
        this.beregnetArbeidsgrad = beregnetArbeidsgrad;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykepengesoknad/Avvik.kt
package no.nav.syfo.controller.domain.sykepengesoknad

import java.io.Serializable

class Avvik : Serializable {
    var arbeidstimerNormalUke: Double? = null
    var arbeidsgrad: Int? = null
    var timer: Double? = null
    var beregnetArbeidsgrad: Int? = null
    fun withArbeidstimerNormalUke(arbeidstimerNormalUke: Double?): Avvik {
        this.arbeidstimerNormalUke = arbeidstimerNormalUke
        return this
    }

    fun withArbeidsgrad(arbeidsgrad: Int?): Avvik {
        this.arbeidsgrad = arbeidsgrad
        return this
    }

    fun withTimer(timer: Double?): Avvik {
        this.timer = timer
        return this
    }

    fun withBeregnetArbeidsgrad(beregnetArbeidsgrad: Int?): Avvik {
        this.beregnetArbeidsgrad = beregnetArbeidsgrad
        return this
    }
}
